mock-core-configs/etc/mock/fedora-rawhide-loongarch64.cfg

```diff
@@ -0,0 +1,4 @@
+config_opts['target_arch'] = 'loongarch64'
+config_opts['legal_host_arches'] = ('loongarch64',)
```

Comment:
This will not work.  Fedora so far doesn't provide repositories for this architecture:
Status code: 404 for https://mirrors.fedoraproject.org/metalink?repo=rawhide&arch=loongarch64

Where are the Fedora (remixed) packages?

Reply:
Oh, that's awkward, maybe have to wait and close this PR.